src/redux/actions/experiments/loadExperiments.js

```diff
@@ -1,65 +1,32 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import endUserMessages from 'utils/endUserMessages';
 
-import {
-  EXPERIMENTS_LOADED,
-  EXPERIMENTS_ERROR,
-  EXPERIMENTS_LOADING,
-} from 'redux/actionTypes/experiments';
+import { EXPERIMENTS_ERROR, EXPERIMENTS_LOADED, EXPERIMENTS_LOADING } from 'redux/actionTypes/experiments';
 
-const toApiV1 = (experimentV2) => {
-  const {
-    id,
-    name,
-    description,
-    samplesOrder,
-    notifyByEmail,
-    processingConfig,
-    createdAt,
-    pipelines,
-  } = experimentV2;
-
-  const experimentV1 = {
-    experimentId: id,
-    projectId: id,
-    description,
-    experimentName: name,
-    processingConfig,
-    createdDate: createdAt,
-    notifyByEmail,
-    sampleIds: samplesOrder,
-    meta: {
-      // This is always 10x and organism so we can just generate them here
-      organism: null,
-      type: '10x',
-      gem2s: pipelines.gem2s ?? undefined,
-      pipeline: pipelines.qc ?? undefined,
-    },
-  };
-
-  return experimentV1;
-};
-
-const loadExperiments = (
-  projectUuid,
-) => async (dispatch) => {
+const loadExperiments = () => async (dispatch) => {
   dispatch({
     type: EXPERIMENTS_LOADING,
   });
 
   try {
-    let data = await fetchAPI(`/v2/experiments/${projectUuid}`);
+    const experiments = await fetchAPI('/v2/experiments');
 
-    data = [toApiV1(data)];
+    experiments.forEach((experiment) => {
+      // eslint-disable-next-line no-param-reassign
+      experiment.sampleIds = experiment.samplesOrder;
+      // eslint-disable-next-line no-param-reassign
```

Comment:
Do you need this reassign?

Reply:
I receive samplesOrder from the api, but we call it sampleIds in the ui, I didn't want to do that rename in this PR too, so this is the best alternative I found

Reply:
oh no, I mean the `// eslint-disable-next-line no-param-reassign` line on line 18

Reply:
resolved in a call